recipes/pom.xml

```diff
@@ -1,192 +1,264 @@
-    <modelVersion>4.0.0</modelVersion>
-
-    <parent>
-        <groupId>io.quarkus</groupId>
-        <artifactId>quarkus-updates-parent</artifactId>
-        <version>1.0.4-SNAPSHOT</version>
-        <relativePath>../pom.xml</relativePath>
-    </parent>
-
-    <artifactId>quarkus-update-recipes</artifactId>
-    <name>Quarkus Updates - Recipes</name>
-
-    <properties>
-        <maven.compiler.release>11</maven.compiler.release>
-
-        <!-- Make sure all these versions are aligned and use a compatible OpenRewrite version -->
-        <!--   https://central.sonatype.com/artifact/org.openrewrite.recipe/rewrite-recipe-bom/2.0.3/versions -->
-        <rewrite-recipe-bom.version>1.19.4</rewrite-recipe-bom.version>
-        <!--   https://central.sonatype.com/artifact/org.openrewrite.maven/rewrite-maven-plugin/5.2.4/versions -->
-        <rewrite-maven-plugin.version>4.46.0</rewrite-maven-plugin.version>
-        <!--   for now, we don't know where to find compatibility information for the Gradle plugin -->
-        <rewrite-gradle-plugin.version>5.40.6</rewrite-gradle-plugin.version>
-
-        <lombok.version>1.18.28</lombok.version>
-    </properties>
-
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>org.openrewrite.recipe</groupId>
-                <artifactId>rewrite-recipe-bom</artifactId>
-                <version>${rewrite-recipe-bom.version}</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
-        </dependencies>
-    </dependencyManagement>
-
-    <dependencies>
-        <!-- rewrite-java dependencies only necessary for Java Recipe development -->
-        <dependency>
-            <groupId>org.openrewrite</groupId>
-            <artifactId>rewrite-java</artifactId>
-            <scope>provided</scope>
-            <exclusions>
-                <exclusion>
-                    <artifactId>antlr-runtime</artifactId>
-                    <groupId>org.antlr</groupId>
-                </exclusion>
-                <exclusion>
-                    <artifactId>javax.json</artifactId>
-                    <groupId>org.glassfish</groupId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-
-        <!-- You only need the version that corresponds to your current
+<project
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>io.quarkus</groupId>
+		<artifactId>quarkus-updates-parent</artifactId>
+		<version>1.0.4-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+	<artifactId>quarkus-update-recipes</artifactId>
+	<name>Quarkus Updates - Recipes</name>
+	<properties>
+		<maven.compiler.release>17</maven.compiler.release>
```

Comment:
Java 17 is used for this change, because it malkes writing the test much easier because of the String block. I think that it is not a strong enough argument to force java version update on the quarkus-recipes.